torch/csrc/jit/passes/alias_analysis.cpp

```diff
@@ -1086,10 +1050,14 @@ bool AliasDb::writesToWildcard(Node* n) const {
   }
   const auto& writes = writeIndex_.at(n);
 
-  // For all writes, check if the written value is a wildcard
-  return std::any_of(writes.cbegin(), writes.cend(), [&](const Value* v) {
-    return mayAliasWildcard(v);
-  });
+  // Are any of these memoryLocs a wildcard element?
```

Comment:
The `writesToWildcard` function seems suspicious given wildcards are not really "special" and just the heap. It seems like it has a single use in `hasUntrackedMutation`. Looking at DCE, it probably also needs a pass to make it understand the wildcard set as the 'heap' rather than special annotation.

Reply:
Good point I'll stack a PR to fix